src/playground/models/user.clj

```diff
@@ -3,10 +3,11 @@
             [clojure.string :as string]))
 
 (spec/def ::username (spec/and string? seq (complement clojure.string/blank?)))
-(spec/def ::password (spec/and string? seq (complement clojure.string/blank?)))
-(spec/def ::name (spec/and string? seq (complement string/blank?)))
+(spec/def ::password ::username)
```

Comment:
a password is not a username, semantically this doesn't make sense.

Please `(spec/def ::present-string  (spec/and string? seq (complement clojure.string/blank?)))` and use it here